main/build.gradle

```diff
@@ -77,7 +77,6 @@ dependencies {
     implementation 'org.locationtech.spatial4j:spatial4j:0.7'
     testImplementation group: 'junit', name: 'junit', version: '4.13'
     testImplementation "com.google.truth:truth:1.0.1"
-    testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
```

Comment:
Nice catch! Let's fully remove it: https://github.com/MobilityData/gtfs-validator/pull/955